src/listener/utils/transforms/plain-text-transformer.js

```diff
@@ -100,8 +103,8 @@ export default class FlowdockTransform {
 
     const out = output(resource).join('\n');
     if (text) {
-      msg.send(out);
-    }
+      msg.send(text);
```

Comment:
Aren't we supposed to use out in the msg.send()?

Reply:
Yes, we are using yet.

Before:
```javascript

if (text) {
      msg.send(out);
}

msg.send(out);
```

After:
```javascript

if (text) {
      msg.send(text);
}

msg.send(out);
```

The bug was fixed, because before the output was being shown 2 times and the text never.